### PR TITLE
Changelog August

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
SUMMARY: None

#### Purpose of change
Update the changelog for the month of May.

#### Describe the solution
tools/generate_changelog.py -f --by-date changelog_08  -t ../repo_token -e 2020-08-01 2020-07-01
tac changelog_08 > changelog_08_rev
tools/merge_summaries.sh changelog_08_rev data/changelog.txt
Then copyedit the results.

#### Additional context
Typos happen, and some of the things I prune might be bigger than I remember or part of some larger set of changes that should go in the changelog, please take a look at not just the final diff, but also the copyedit commit to check for missed entries.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Nothing you pruned looks too important to me.

File: data/changelog.txt [hunk 1]
```diff
@@ -44,11 +50,14 @@ Allow the of use grappling hooks on ledges.
 Make bugs much more mutated.
 Reformats the 'chat with a friend' feature in NPCs to be more flexible, and to provide some more lore insights.
 Adds 'What the hell happened' stories to NPCs, keyed off backgrounds.
+Feral Humans with blob psychosis.
+Allow plank and branch bundles to be made from cordage, vines, or wire
```
Comment (ZhilkinSerg) [inline data/changelog.txt:54]:
```suggestion
Allow plank and branch bundles to be made from cordage, vines, or wire.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -95,9 +105,15 @@ Fix segfault on harvest.
 Fix loading of legacy characters without bodies.
 Fix explosions not affecting terrain.
 Do not process held item twice.
+Fix crash when trying to reloading item inside advanced inventory
+Blobs are slimes. name them such.
+Fix freezing to death when wet.
+Fix weapon techniques not triggering.
+Prevent some obscure crashes by adding nullptr checks to unsafe pointer accesses
```
Comment (ZhilkinSerg) [inline data/changelog.txt:112]:
```suggestion
Prevent some obscure crashes by adding nullptr checks to unsafe pointer accesses.
```

File: data/changelog.txt [hunk 3]
```diff
@@ -135,6 +151,15 @@ Enforce use of points as local variables.
 Add table.py to make tables of JSON data.
 Add a guide for intermediate mapgen help.
 Options: remove the option to not use z-levels.
+Provide point types with coordinate system type safety.
+Move weather from being hardcoded into json.
+Update Catch2 to 2.13.0
```
Comment (ZhilkinSerg) [inline data/changelog.txt:156]:
```suggestion
Update Catch2 to 2.13.0.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -144,13 +169,18 @@ Makes json linter feedback more readable.
...
+Improve use of precompiled headers.
+compose.py: allow default sized tilesheets to specify offsets.
 
 ## I18N and A11Y:
 Fix translation of martial art initate message and computer translation.
 Add support for specifying identical singular and plural forms using `str_sp`.
 Remove name_plural and replace it with str_pl.
 Add helper for localized sorting.
 Record locales in debug.log to assist with i18n debugging.
+Add translator glossary and explain etymology of Exodii
```
Comment (ZhilkinSerg) [inline data/changelog.txt:183]:
```suggestion
Add translator glossary and explain etymology of Exodii.
```

**Pull request merged**